shackleton/src/RichList.java

```diff
@@ -19,9 +19,6 @@
 import snowblossom.client.GetUTXOUtil;
 
 import java.text.DecimalFormat;
-import com.google.protobuf.ByteString;
```

Comment:
I suspect none of these changes to RichList should go in, looks like a merge error.